examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>gates <strong>in </strong>your circuit. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>